### ISSUE TITLE
Ability to save private/hidden sketches.

### ISSUE THREAD

**Issue by AnnaUdovichenko** [Sketch-sh/sketch-sh#144]
Now all links to sketches can be found through user's profile. It would be really great to be able to share a sketch with somebody by URL without showing it to everybody (or just play with code, edit it from different PCs etc. while keeping it private). Is it possible to add such a feature?

Thank you!

**Comment by kyldvs** [comment]
Yeah, I don't think sketches should be visible on your profile by default unless you opt into that

**Comment by thangngoc89** [comment]
I agree with this. I will introduce a publish mode in the future so only published sketches will be listable
